app/views/case_studies/index.html.haml

```diff
@@ -3,13 +3,19 @@
     var pH = $('#create-btn').parent().parent().height()
     var eH = $('#create-btn').height()
     $('#create-btn').css('margin-top', (pH - eH)/2)
+    $('#hidden').click(function() {
+      $("#hiddenCaseStudies").toggle(this.checked);
+    });
   });
 .container-fluid
   .row
     .col-md-2
     .col-md-10.left-pad
       .col-md-6
         %h3 Featured Case Studies
+        -if current_user.admin?
```

Comment:
I think editors should be able to view hidden assets(methods and cases) because editors can make assets hidden via the edit form. Wouldn't make much sense for a user to be able to hide a method and then have it disappear into the void until an admin rescues it. Since editor is not a superset of admin, I'd rewrite this line as `-if current_user.admin? || current_user.editor?` so we don't exclude admins